### DISCUSSION TITLE
Plot function freezes my IDE or browser when I run it and click on the plot

### DISCUSSION THREAD

**Discussion by Precisionguided** [kernc/backtesting.py#425] [Q&A]
Any idea what's causing it? I have a relatively powerful CPU and GPU but when I plot the results of the simple SMA strategy and I click anywhere on the graph, it gets really slow then crashes vscode or my browser, depending on whether I'm using a jupyter notebook or a python script.

**Comment by kernc** [comment]
Do you maybe use [`Backtest.plot(resample=False)`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Backtest.plot)? We resample the output to 10000 candles by default exactly so as to reduce browser burden. How many candles and indicators are we talking here?

**Reply by Precisionguided** [reply]
I don't, but when I researched further it looks like it's because the resample won't work on non datetime indexes. I have mine converted to the format like 2021-06-20 19:15:00, but not sure how I get it to be read as datetime, if that makes sense.

**Reply by Precisionguided** [reply]
Was able to fix it by using the ff:

`eth_df=pd.read_csv('ftx-ETH-PERP-1m-2020-01-01.csv',parse_dates=['Timestamp'])
eth_df['Timestamp']=eth_df.set_index(pd.to_datetime(eth_df['Timestamp'],unit='ms',origin='unix'),inplace=True)`

converting what was in unix timestamp to datetime and to an index and that improved the performance of the plot.

Thanks.
